.github/workflows/ci-cd.yml

```diff
@@ -13,7 +13,7 @@ on:
     - patchback/backports/**  # Patchback always creates PRs
     - pre-commit-ci-update-config  # pre-commit.ci always creates a PR
   pull_request:
-    ignore-paths:  # changes to the cron workflow are triggered through it
+    paths-ignore:  # changes to the cron workflow are triggered through it
```

Comment:
I extracted this into `main`. I have this typo across a bunch of repos, actually. Good catch!

@sirosen any insight into why jsonschema didn't catch this?